Assets/Scripts/State/GameState.cs

```diff
@@ -7,6 +7,8 @@ public class GameState : MonoBehaviour
 {
     public static GameState Instance { get; private set; }
     public GridTile[,] WallMap { get; private set; }
+    public WallBehaviour[,] WallObjects { get; private set; }
```

Comment:
preco wall object je ako zvlast 2. 2d array map 
preco to nieje sucastou gridTile 
grid tile obsahuje info ci to je stena alebo nie, cize aj tak ju musis upravovat pri vytvarani aj pri niceni

Reply:
vies preco, vysvetloval som ti to dnes 20 minut